pkg/apis/awsprovider/v1alpha1/awsmachineproviderconfig_types.go

```diff
@@ -74,6 +74,10 @@ type AWSMachineProviderSpec struct {
 	// +optional
 	KeyName string `json:"keyName,omitempty"`
 
+	// RootDeviceSize is the size of the root volume.
+	// +optional
+	RootDeviceSize int64 `json:"rootDeviceSize,omitempty"`
```

Comment:
I think you should duplicate this change in the CRD yaml: https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/master/config/crds/awsprovider_v1alpha1_awsmachineproviderspec.yaml

Reply:
ah! forgot to run `make generate-crds`. thanks Seth! updated.